### PR TITLE
Update user event to new version

### PR DESCRIPTION
**Pull request by hieudion**
Because Zendesk update the UserEvent endpoint to version 2 and deprecated the old endpoint.
We should update in our plugin too.

### CODE REVIEW

File: src/main/java/org/embulk/input/zendesk/stream/paginator/sunshine/UserEventSpliterator.java [hunk 1]
```diff
@@ -13,6 +15,8 @@
 {
     private String userID;
 
+    private static final Logger logger = LoggerFactory.getLogger(UserEventSpliterator.class);
+
```
Comment (trung-huynh) [inline src/main/java/org/embulk/input/zendesk/stream/paginator/sunshine/UserEventSpliterator.java]: is logger being used somewhere?

**Review by trung-huynh — APPROVED** [review]
left a comment, general looks good to me.

File: src/main/java/org/embulk/input/zendesk/stream/paginator/sunshine/UserEventSpliterator.java (hunk 1 above)
Comment (hieudion) [reply src/main/java/org/embulk/input/zendesk/stream/paginator/sunshine/UserEventSpliterator.java]: Fixed

**Comment by hieudion** [comment]
@ylzen Could you help to review this PR?

**Review by ylzen — APPROVED** [review]
Reviewed for Data Exfiltrations.

**Pull request merged**